addon/helpers/equal.js

```diff
@@ -1,3 +1,13 @@
 export function equalHelper(params) {
-  return params[0] === params[1];
+    let len = params.length;
```

Comment:
Can't we go even further and remove indexes? I mean something like [reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce) or [every](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every)?